Library/Formula/kobalt.rb

```diff
@@ -0,0 +1,46 @@
+class Kobalt < Formula
+  desc "Build system"
+  homepage "http://beust.com/kobalt"
+  url "https://github.com/cbeust/kobalt/releases/download/0.633/kobalt-0.633.zip"
```

Comment:
Kobalt is written in Kotlin actually but same thing: the zip file contains a jar which is cross platform.